### ISSUE TITLE
Free Theme gives 404 Not Found error

### ISSUE THREAD

**Issue by Phelanopsis** [primefaces/primeng#4207]
When we select any themes under "Free" category, we receive the below error:

Request URL:http://localhost:4200/assets/components/themes/rocket/theme.css
Request Method:GET
Status Code:404 Not Found

**Comment by sudheerj** [comment]
Are you talking about showcase? Did you build the CSS as mentioned in build from source wiki? Please use forums for questions.

**Comment by Phelanopsis** [comment]
Yes, its about showcase. When tried to build CSS , got the below error even though node-sass was successfully installed. 
I will ask this issue on forum. Thanks.

sass : The term 'sass' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was
included, verify that the path is correct and try again.
At line:1 char:1
+ sass --update src/assets/components
+ ~~~~
    + CategoryInfo          : ObjectNotFound: (sass:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

**Comment by cagataycivici** [comment]
Please use forum for cases like these.
